contracts/OpenSTUtility.sol

```diff
@@ -104,8 +106,13 @@ contract OpenSTUtility is Hasher, OpsManaged, STPrimeConfig {
     /// chainId of the current utility chain
     uint256 public chainIdUtility;
     address public registrar;
+    uint256 public blocksToWaitShort;
+    uint256 public blocksToWaitMedium;
```

Comment:
similarly, the `blocksToWaitMedium` is not needed in OpenSTValue and OpenSTUtility contracts